crates/maybenot-ffi/Makefile

```diff
@@ -0,0 +1,30 @@
+CARGO ?= cargo
+TARGET ?=
+PROFILE ?= release
+DESTINATION ?= .
+
+CARGOFLAGS += --locked
+TARGET_DIR := ../../target/$(TARGET)/$(PROFILE)
+
+ifeq ($(PROFILE), release)
+	CARGOFLAGS += --release
+endif
+
+ifneq ($(TARGET),)
+	CARGOFLAGS += --target
+	CARGOFLAGS += $(TARGET)
+endif
+
+.PHONY: clean
+
+# copy the library to the final destination, and strip the _ffi part
+$(DESTINATION)/libmaybenot.a: $(TARGET_DIR)/libmaybenot_ffi.a
+	cp $^ $@
+
+# build the static library
+$(TARGET_DIR)/libmaybenot_ffi.a: src/*.rs Cargo.toml build.rs cbindgen.toml
+	RUSTFLAGS="-C metadata=rustls-ffi" ${CARGO} build $(CARGOFLAGS)
+
+clean:
+	rm -f $(DESTINATION)/libmaybenot.a
+	cargo clean
```

Comment:
Regarding my above comment about `$CARGO_TARGET_DIR`. If that variable is set to an absolute path (what I have) then this command will clear the entire cargo cache for that user :sweat_smile: Maybe that's OK, maybe it's just my setup being too weird :upside_down_face:

Reply:
Hmm, hadn't thought about CARGO_TARGET_DIR. Cleaning the entire cargo cache is definitely not desired, but `make clean` *should* ideally clean up the build artefacts for maybenot_ffi... I'll see if I can come up with a better solution.

Reply:
Maybe it's overkill trying to handle that! I should maybe not have opened that can of worms. Removing the entire cargo cache is not a dangerous operation. Worst case it can be somewhat of a time sink, but nothing worse.